docs/DeveloperGuide.md

```diff
@@ -1,38 +1,253 @@
+|v1.0|user| add, delete, edit, and list my expenses                               | manage my expenses                               |
+|v1.0|user| set and view my budget                                                | set expectation of how much money I should use   |
+|v1.0|user| view how much of the budget I spend                                   | manage and change my spending habit as necessary |
+|v1.0|user| view all comments that I can enter                                    | get help on the features if necessary            |
+|v2.0|user| list monthly expenses, income, and budget                             | refer to financial status in previous months     |
+|v2.0|user| set the category of income and expenses, and list entries by category | refer to expenses and income by category         |
+
+## Appendix: Requirements
+
+### Product scope
+
+**Target user profile**:
+
+- BrokeMan is suitable for students who have to work with tight budgets
+- students who want to use their money efficiently
+- students who want to minimize their spending.
+- students who can type fast
+- prefer typing to mouse interactions
+- is reasonably comfortable using CLI apps
+
+**Value proposition**:
+
+- Manage income and expenses faster than a typical mouse/GUI driven app
+- The program will offer visualization of the user's incomes, expenses and budget, allowing them to recap and be mindful
+  about their financial status. The project will allow division of budget into multiple subcategories of expenses. In
+  essence, the program sets students up for a better financial future.
+
+### User Stories
+
+Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unlikely to have) - `*`
+
+| Priority | Version | As a ... | I want to ...                             | So that I can ...                                |
+|----------|---------|----------|-------------------------------------------|--------------------------------------------------|
+| `* * *`  | v1.0    | user     | add, delete, edit, and list my income     | manage my income                                 |
+| `* * *`  | v1.0    | user     | add, delete, edit, and list my expenses   | manage my expenses                               |
+| `* * *`  | v1.0    | user     | set and view my budget                    | set expectation of how much money I should use   |
+| `* * *`  | v1.0    | user     | view how much of the budget I spend       | manage and change my spending habit as necessary |
+| `* * *`  | v1.0    | user     | view all command that I can enter         | get help on the features if necessary            |
+| `* * *`  | v2.0    | user     | list monthly expenses, income, and budget | refer to financial status in previous months     |
+| `* * *`  | v2.0    | user     | save all my income and expenses entered   | so that I can refer to it next time I return     |
```

Comment:
This is a second User Stories table that is almost similar to the first one above, is this supposed to be combined or meant to be an additional table? Can consider providing just 1 table to reduce confusion 👍